qa/standalone/misc/test_failure.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
this test fails http://pulpito.ceph.com/kchai-2017-07-30_00:01:51-rados-wip-kefu-testing-distro-basic-smithi/1460917/. because the test of `rados/standalone/misc.yaml` runs every executable under `qa/standalone/misc/`. and the executables under that directory are not launched by `qa/run-standalone.sh`.